src/main/java/com/SAS/teamManagenemt/TeamManagement.java

```diff
@@ -18,6 +21,52 @@ public TeamManagement(UserController userController) {
         this.userController = userController;
     }
 
+    /**
+     * This function allows the team owner to add an asset to the team. Assets are facility, player, coach.
+     * @param assetType
+     * @param team
+     * @return the created asset
+     */
+    public TeamAsset AddAssetToTeam (String assetType, Team team) {
+        TeamAsset asset = null;
+        //create the asset
+        switch (assetType) {
+            case "Player":
+                asset = (Player) userController.createUser(null, null, null, UserType.PLAYER, true);
```

Comment:
You create the player with null fields and later his details will be edited?

Reply:
yes